test/tstRay.cpp

```diff
@@ -172,7 +172,7 @@ BOOST_AUTO_TEST_CASE(intersects_box)
   {                                                                            \
     float t0;                                                                  \
     float t1;                                                                  \
-    constexpr auto inf = KokkosExt::ArithmeticTraits::infinity<float>::value;  \
+    auto const inf = KokkosExt::ArithmeticTraits::infinity<float>::value;      \
```

Comment:
You commented about aggregate initialization issues in constant expressions below but what was the deal with variable templates?

Reply:
@AnhBe0 can you post the error message, I don't remember the issue exactly.

Reply:
Yes , here is the error message. 
![image](https://github.com/arborx/ArborX/assets/116529205/1ebf3899-0dbe-4d79-8246-cb1336d2e5bb)

Reply:
Which one corresponds to the line Damien pointed out? I don't see an error in line 175

Reply:
Sorry it's the right line. I forgot we are using a macro...